packages/library/src/utils/fetchReceiverDrawIdToPush.ts

```diff
@@ -0,0 +1,112 @@
+import { Contract } from '@ethersproject/contracts';
+
+const debug = require('debug')('pt-autotask-lib');
+
+/**
+ * Fetches the latest draw ids from the beacon and receiver chain draw buffers and returns the next draw id to push to the receiver chain.
+ * PoolTogether v4 v1.3.0 Architecture
+ * @param drawBufferBeaconChain
+ * @param prizeDistributionBufferBeaconChain
+ * @param prizeDistributionBufferReceiverChain
+ * @returns
+ */
+export async function fetchReceiverDrawIdToPush(
```

Comment:
We can remove this file, L2s will work the same way than beacon chains now that we either compute the draw on-chain or bridge it.